### PR TITLE
T163: create NavBar

### PR DESCRIPTION
**Pull request by kdata12**
## Description 

Create NavBar version 1.0

https://user-images.githubusercontent.com/106122976/236643291-f6a1b2b0-1d49-4998-a722-030fa2db4cf9.mp4

edit: add 2 separate navbar, one for logged in user and the other for logged out user

## Types of changes

What types of changes does your code introduce to WeWrap?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have made sure that eslint was passed locally
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/src/components/NavBar/NavBar.module.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.signup span {
+  color: var(--light-yellow);
+}
+
+.signup a {
+  background-color: var(--green);
+}
+
+
```
Comment (JustinChen-at) [inline frontend/src/components/NavBar/NavBar.module.css]: extra new line here, rm one

File: frontend/src/components/loginForm.tsx [hunk 2]
```diff
@@ -20,7 +31,7 @@ export const LoginForm = (): JSX.Element => {
       }, {
         withCredentials: true
       })
-      navigate('/tempLandingPage')
+      window.location.href = '/hub';
```
Comment (JustinChen-at) [inline frontend/src/components/loginForm.tsx:34]: should this be using navigate?

**Review by JustinChen-at — APPROVED** [review]
small nits/questions - lgtm!

File: frontend/src/components/loginForm.tsx (hunk 2 above)
Comment (kdata12) [reply frontend/src/components/loginForm.tsx:34]: I tried using the navigate hook but it didn't reload and update the navbar. I used window.location.href to make browser do a full reload so the navbar can update based on user logged in state

**Review by justinpchen94 — APPROVED** [review]

**Pull request merged**